stories/topics/proc-aws-backup-and-restore.adoc

```diff
@@ -159,7 +159,7 @@ $ docker pull $IMAGE
 +
 [NOTE]
 =====  
-The `ansible-on-clouds-ops` image tag should match the version of your foundation deployment. For example, if your foundation deployment version is 2.2.20230215-00, pull the `ansible-on-clouds-ops` image with tag 2.2.20230215-00.
+The `ansible-on-clouds-ops` image tag should match the version of your foundation deployment. For example, if your foundation deployment version is 2.2.20230215, pull the `ansible-on-clouds-ops` image with tag 2.2.20230215.
```

Comment:
from
foundation deployment version is 2.2.20230215
to
foundation deployment version is 2.2.20230215-00

Reply:
updated

Reply:
Actually I'm not sure right now if this is for upgrade section or backup/restore section?  Need to look more

Reply:
After further investigation, I dont think this page is linked up properly to display as it was replaced with other backup and restore pages which are configured correctly @chrisahl.
This is likely from our 2.2 doc and isnt linked properly to display in our 2.3 UI, but the 2.3 b&r and upgrade pages already have correct content if im notmistaken.